common/include/pcl/impl/point_types.hpp

```diff
@@ -1045,37 +927,29 @@ namespace pcl
     */
   struct PointXYZRGBNormal : public _PointXYZRGBNormal
   {
-    inline PointXYZRGBNormal (const _PointXYZRGBNormal &p)
-    {
-      x = p.x; y = p.y; z = p.z; data[3] = 1.0f;
-      normal_x = p.normal_x; normal_y = p.normal_y; normal_z = p.normal_z; data_n[3] = 0.0f;
-      curvature = p.curvature;
-      rgba = p.rgba;
-    }
+    inline constexpr PointXYZRGBNormal (const _PointXYZRGBNormal &p) :
+      PointXYZRGBNormal {p.x, p.y, p.z, p.r, p.g, p.b, p.normal_x, p.normal_y, p.normal_z, p.curvature} {}
```

Comment:
seems like there's no `a` in this structure, are you sure there's actually an error here?

Reply:
`_PointXYZRGBNormal` has `PCL_ADD_UNION_RGB`, which defines `a` and `rgba`. Previously, it was copied via `rgba = p.rgba;`. But now, it is always set to 255 in the other constructor

Reply:
right, i got confused, because this type's behaviour is different from other point types: here ctors delegation chain is different. i've only managed to solve this with additional constructor introduction (introduction of additional default params is another way, but that would require rearrangement of params and would break logical grouping)